client/parties/actions.ts

```diff
@@ -0,0 +1,121 @@
+import { PartyPayload, PartyUser } from '../../common/parties'
+import { BaseFetchFailure } from '../network/fetch-action-types'
+
+export type PartiesActions =
+  | InviteToPartyBegin
+  | InviteToPartySuccess
+  | BaseFetchFailure<'@parties/inviteToParty'>
+  | RemovePartyInviteBegin
+  | RemovePartyInviteSuccess
+  | BaseFetchFailure<'@parties/removePartyInvite'>
+  | DeclinePartyInviteBegin
+  | DeclinePartyInviteSuccess
+  | BaseFetchFailure<'@parties/declinePartyInvite'>
+  | AcceptPartyInviteBegin
+  | AcceptPartyInviteSuccess
+  | BaseFetchFailure<'@parties/acceptPartyInvite'>
+  | AddInvite
+  | RemoveInvite
+  | Init
+  | Invite
+  | Uninvite
+  | Decline
+  | Join
+  | Leave
+
+export interface InviteToPartyBegin {
+  type: '@parties/inviteToPartyBegin'
```

Comment:
These `Begin` actions should *always* include information about the action that is being performed. "I'm about to do a party invite" is really useless information if you don't know *who* is being invited, there's nothing you can do with that info.

Apply throughout